svthalia/concrexit
# Improve summary reward functionality

Closes #1096 

### Summary
Improve summary reward functionality, adding fields to the admin for information and an explanation to the frontend

### How to test
Steps to test the changes you made:
1. Go to the course overview and check the texts
2. Go to the admin and check that the summary form can still be used to create and update. And only in update mode shows the uploader information
